### ISSUE TITLE
Replace real anti cheat logs with fake logs

### ISSUE THREAD

**Issue by Koles1910** [grimaldello/the-crims-italian-bot-v2#57]
add the ability to set fake mouse tracking

**Comment by Koles1910** [comment]
when are you planning to release the improved edition?

**Comment by grimaldello** [comment]
This approach can be implemented, but honestly it is a "last resort".

It has low priority for me.

If anybody want to implement it and contribute to the bot, I'm glad for that.

**Comment by zoiodelula** [comment]
How do i can contribute with that?

**Comment by grimaldello** [comment]
I think that one way implement it is to intercept the request to the server with events logs.
Once intercepted, replace the real events logs with a set of fake logs.
These fake logs must have the same structure with similar values of the original ones but could be a greater number of events.
Or as alternative, duplicates (or add some) events and change only the values of small amount of number.

In addition to this, I found that maybe the number of mouse events (and so number of logs) could depend on this: https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/getCoalescedEvents
So basically a group of mouse events can be "merged" in a single event (for performance)
